assisted-events-scrape/workers/cluster_events_worker.py

```diff
@@ -174,9 +176,13 @@ def _cluster_checksum(self, doc: dict) -> dict:
             doc_copy["hosts"].sort(key=by_id)
         return get_dict_hash(doc_copy)
 
+    def _enrich_cluster(self, doc: dict):
+        doc["cluster_state_id"] = str(uuid5(CLUSTER_STATE_ID_NAMESPACE, self._cluster_checksum(doc)))
+
 
 def by_id(item: dict) -> str:
-    return item.get("id", None)
+    # ID should always be there, if not consider it empty string
```

Comment:
transform to a docstring instead of a comment
btw, is it that ``item`` should always have the ``id`` property or that we should always return some kind of value for the function that calls ``by_id``?
if the first I would expect us to just use ``item["id"]`` (and let it raise if our expectations were wrong or if the component filling-in ``item`` did it wrongly)

Reply:
This is used to sort the host list in the cluster objects, therefore:
* we always expecte "id" field (in hosts objects), however if it doesn't show up for any reason I wouldn't break the whole thing
* return value must be always the same type, as it is used for comparison for ordering the list